### DISCUSSION TITLE
Where can I find the license terms?

### DISCUSSION THREAD

**Discussion by BillButter** [photoprism/photoprism#2245] [Q&A] [answered]
Clicking on the AGPL button takes me to a 404. Have they published their license in full yet?

**Comment by graciousgrey** [comment, marked as answer]
Good catch, need to update the link --> https://docs.photoprism.app/license/agpl/

**Reply by lastzero** [reply]
Fixed in https://github.com/photoprism/photoprism/commit/68a441ecdfcc1a9f9ca56555fd60af8c6ffecd9f :+1:
